src/pages/hegel/reference/the-mechanical-process/the-formal-mechanical-process-section-1.mdx

```diff
@@ -0,0 +1,243 @@
+speaking, there is no change from one thing to something different. It is quite
+simply the identical determinateness that exists between `mechanical objects` –
+a kind of ether of determinateness that connects all objects together but does
+not affect them beyond connecting them.
+
+### Communication Between Objects
+
+Hegel then embarks on a remark that helpfully unpacks what he has in mind with
+the logical determination of `communication`. He writes:
+
+> _Spiritual communication_, which moreover takes place in that element which is
+> the universal in the form of universality, is explicitly an _ideal_ relation
+> in which _a determinateness continues_ itself from one person into another
+> unimpaired, and universalizes itself without any alteration whatever-as a
+> scent freely spreads in the unresisting atmosphere. But even in communication
+> between material objects, their determinateness _spreads_, so to speak, in a
+> similarly ideal manner; personality is an infinitely more intense
+> _impenetrability_ [*Harte*] than objects possess (Hegel 1991, 716). The above
+> paragraph provides a series of concrete instances of `communication`.
+> Crucially, the examples given are all of the kind of communication that occurs
+> between objects; Hegel makes the point to distinguish the kind of
+> communication between objects to something like personality, which “is an
+> infinitely more intense _impenetrability_ [*Harte*] than objects possess”
+> (Hegel 1991, 716). This is an excellent opportunity to look closely at Hegel’s
+> examples as they provide a window into how he thinks about the real-world
+> instances of the logical development. If we spend a bit of time unpacking
+> Hegel’s examples and scrutinising their validity, we will get a clearer
+> understanding of how he thinks of ontology.
+
+Beginning with Hegel’s first example, that of a scent diffusing in an atmosphere
+where there is no restriction on the diffusion of the scent. Why does Hegel
+think that this is a helpful example of how `communication` is exemplified in
+everyday life? At first glance, it makes sense. The way a scent freely diffuses
+itself around an atmosphere so that the whole atmosphere smells of the scent is
+reminiscent of the identical determinateness of the `mechanical object` that
+leads to it becoming a universal. However, how _exactly_ are we supposed to
+understand this? Is the mechanical process at work between the scented molecules
+and the non-scented molecules, or is it just between the scented molecules? To
+pose this question is, I think, to overstep the conceptual restrictions of
+Mechanism and mechanistic thought. The distinction between a scented molecule
+and a non-scented molecule does not obtain within a mechanistic framework
+because, as `mechanical objects`, they do not have different determinations
+(_i.e._ one is scented and the other is non-scented). Quite simply, scented
+molecules and non-scented molecules have an identical determinateness.
```

Comment:
Okay, but I'm not left with a better sense of what communication means through this example. Not sure if it's the example itself or your explanation. Perhhaps it would be good to add some conclusion as to what benefits we have gained to our understanding through this example?

Reply:
I've gone down a bit of a rabbit-hole, but I think it works.

"What the above example provides us with, is a way of thinking about the abstract
logical structure of `communication` in a concrete way. The diffusion of the 
scent is the unimpeded movement of the identical determinateness of `mechanical
objects` and the fact that a scent will diffuse throughout a room and thus 
universalises itself within the room - in the sense that the scent can be smelt
throughout the room. As such, Hegel's account of `communication` gives a philosophical
explanation for why a scent diffuses through a room. Note, a philosophical
explanation does not seek to provide the material causes for why a scent
diffuses in a room. It explains what the (onto-)logical reasons for scent
diffusion are. These ontological reasons are: all of the particles in a room 
are treated as the same, they are all `mechanical objects`; and the diffusion 
of the scent, then, is explained by the fact that the particles share an identical 
determinateness - they are identical. 

A final note. This does not supplant a materialist explanation. Nor is it merely
a description of what is going on. The ontological reasons exist side-by-side the
materialist causes. The ontological reasons provide the conditions under which
the material causes can then find expression in the world. It is only because
particles `mechanical objects` are identical to each other that the diffusion
of a scent in external reality can occur. Because, it is a philosophical position
to conceptualise all objects as identical - it is not self-evident from the diversity 
of objects that we see around us."

Thoughts?

Reply:
Yeah, that's much better. Thank you! 🤩

Reply:
Thanks for the question. It prompted me to have to think about this cool topic more.